app/views/tag/_tags.html.erb

```diff
@@ -52,3 +52,11 @@ $(".pop").popover({ trigger: "manual" , html: true, animation:false})
 });
 
 </script>
+
+<script>
+$('.tag-delete').click(
+  function(){
+    $('.popover').remove();
+  }
+);
+</script>
```

Comment:
Although, the code looks good. Can we add the code in the above <script> tag? Thanks!

Reply:
Sure @gauravano  !!!